openshift/radargun-template.json

```diff
@@ -391,6 +401,12 @@
          "displayName": "The namespace attribute from openshift plugin",
          "name": "NAMESPACE",
          "required": false
+      },
+      {
+         "description": "The RadarGun full image name published to the registry",
+         "displayName": "The RadarGun full image name published to the registry",
+         "name": "RADARGUN_FULL_IMAGE_NAME",
```

Comment:
Personally, I'd go with plain `RADARGUN_IMAGE` as 'image' is pretty well understood terminology in the OpenShift context. Technically it doesn't have to be full name if the image will exist on DockerHub you could use just `radargun`. 

Or even just `IMAGE` since it's RadarGun template and no other image is used within it.